### PR TITLE
Make admin user instructions a touch more obvious.

### PR DESCRIPTION
**Pull request by Leopere**

### CODE REVIEW

**Comment by Leopere** [comment]
I've added some clarification and highlights for steps that seemed less than self evident on my first attempt at installation.

The AI automatically inserted some numbered bullet points where otherwise there were none and the Grammarly software fixed a pile of the grammatical aberrations caused by myself, the AI, and previous documenters.  This rewrite doesn't accommodate for any new use cases or anything interesting its purely just a clarity and readability cleanup.

File: README.md [hunk 1]
```diff
@@ -2,111 +2,78 @@
 
 [![Discord](https://img.shields.io/discord/559159668438728723.svg?logo=discord)](https://discord.gg/2TsNFKt)
 
-**stash-box is Stash App's own OpenSource video indexing and Perceptual Hashing MetaData API server for porn.**
+Stash-box is an open-source video indexing and metadata API server for porn developed by Stash App. The purpose of stash-box is to provide a community-driven database of porn metadata, similar to what MusicBrainz does for music. The submission and editing of metadata should follow the same principles as MusicBrainz. [Learn more here](https://musicbrainz.org/doc/Editing_FAQ). Installing Stash-box will create an empty database for you to populate.
```
Comment (DogmaDragon) [inline README.md:5]: Removed the phrase `perceptual hashing`. I would say that it's worth mentioning as it's one of the core aspects.

File: README.md [hunk 2]
```diff
@@ -2,111 +2,78 @@
...
-A docker-compose file for production deployment can be found [here](docker/production/docker-compose.yml). Traefik can be omitted if you don't need a reverse proxy.
-
-Alternatively, if postgresql is already available, stash-box can be installed on its own from [dockerhub](https://hub.docker.com/r/stashapp/stash-box).
+You can find a `docker-compose` file for production deployment [here](docker/production/docker-compose.yml). You can omit Traefik if you don't need a reverse proxy.
 
-# Bare-metal Install
+If you already have PostgreSQL installed, you can install stash-box on its own from [Docker Hub](https://hub.docker.com/r/stashapp/stash-box).
 
-Stash-box supports macOS, Windows, and Linux.  
+# Bare-Metal Install
```
Comment (DogmaDragon) [inline README.md]: Would change it to `# Bare-metal install` to keep capitalization consistent with other headings.

File: README.md [hunk 3]
```diff
@@ -2,111 +2,78 @@
...
-
-The `sslmode` parameter is documented in the [pq documentation](https://godoc.org/github.com/lib/pq). Use `sslmode=disable` to not use SSL for the database connection. The default value is `require`.
+1. Run `make` to build the application.
+2. Stash-box requires access to a PostgreSQL database server. Suppose stash-box doesn't find a configuration file (defaults to `stash-box-config.yml` in the current directory). In that case, it will generate a default configuration file with a default PostgreSQL connection string (`postgres@localhost/stash-box?sslmode=disable`). You can adjust the connection string as needed.
+3. The database must be created and available. If the PostgreSQL user is not a superuser, run `CREATE EXTENSION pg_trgm; CREATE EXTENSION pgcrypto;` by a superuser before rerunning Stash-box. If the schema is not present, it will be created within the database.
+4. The `sslmode` parameter is documented [here](https://godoc.org/github.com/lib/pq). Use `sslmode=disable` to not use SSL for the database connection. The default is `require`.
+5. After ensuring the database connection and availability, rerun Stash-box.
+#### Schema migrations and initial Admin user.
```
Comment (DogmaDragon) [inline README.md]: Would remove the `.` from the heading.

File: README.md [hunk 4]
```diff
@@ -2,111 +2,78 @@
...
+The second time that stash-box is run, stash-box will run the schema migrations to create the required tables. It will also generate a `root` user with a random password and an API key. These credentials are printed once to stdout and are not logged. The system will regenerate the root user on startup if it does not exist. You can force the system to create a new root user by deleting the root user row from the database and restarting Stash-box. You'll need to capture the console output with your Admin user on the first successful StashDB executable start. Otherwise, you will need to allow Postgres to re-create the database before it will re-post a new `root` user.
 
-After ensuring the database connection string is correct and the database server is available, the stash-box executable may be rerun.
+# Stash-box CLI and Configuration
```
Comment (DogmaDragon) [inline README.md]: Would change it to `# Stash-box CLI and configuration` to keep capitalization consistent with other headings.

File: README.md [hunk 5]
```diff
@@ -2,111 +2,78 @@
...
 
-The graphql playground and cross-domain cookies can be disabled by setting `is_production: true`.
+- Host: `0.0.0.0`
+- Port: `9998`
 
-### API keys and authorisation
+You can change these defaults if needed. For example, if you want to disable the Graphql playground and cross-domain cookies, you can set `is_production` to `true`.
 
-A user may be authenticated in one of two ways. Session-based management is possible by logging in via `/login`, passing form values for `username` and `password` in plain text. This sets a cookie which is required for subsequent requests. The session can be ended with a request to `/logout`.
+## API Keys and Authorization
```
Comment (DogmaDragon) [inline README.md]: Would change it to `## API keys and authorization` to keep capitalization consistent with other headings.

File: README.md [hunk 6]
```diff
@@ -2,111 +2,78 @@
...
-| `s3.secret ` | (none) | Secret Access key used for authentication. |
-| `s3.max_dimension` | (none) | If set, a resized copy will be created for any image whose dimensions exceed this number. This copy will be served in place of the original.
-| `phash_distance` | 0 | Determines what binary distance is considered a match when querying with a phash fingeprint. Using more than 8 is not recommended and may lead to large amounts of false positives. **Note**: The [pg-spgist_hamming extension](#phash-distance-matching) must be installed to use distance matching, otherwise you will get errors. |
-| `favicon_path` | (none) | Location where favicons for linked sites should be stored. Leave empty to disable. |
-| `draft_time_limit` | (24h) | Time, in seconds, before a draft is deleted. |
-| `profiler_port` | 0 | Port on which to serve pprof output. Omit to disable entirely. |
-| `postgres.max_open_conns` | (0) | Maximum number of concurrent open connections to the database. |
-| `postgres.max_idle_conns` | (0) | Maximum number of concurrent idle database connections. |
-| `postgres.conn_max_lifetime` | (0) | Maximum lifetime in minutes before a connection is released. |
```
Comment (DogmaDragon) [inline README.md]: A whole section is being removed that explains what different configuration keys in `stash-box-config.yml` means.

File: README.md [hunk 7]
```diff
@@ -2,111 +2,78 @@
...
 
-This command would need customizing for your environment.  [This link](https://stackoverflow.com/questions/10175812/how-to-create-a-self-signed-certificate-with-openssl) might be useful.
 
-Once you have a certificate and key file name them `stash-box.crt` and `stash-box.key` and place them in the directory where stash-box is run from. Stash-box detects these and starts up using HTTPS rather than HTTP.
+You might need to modify the command for your specific setup. You can find more information about creating a self-signed certificate with OpenSSL [here](https://stackoverflow.com/questions/10175812/how-to-create-a-self-signed-certificate-with-openssl).
+
+Once you've generated the certificate and key pair, make sure they're named `stash-box.crt` and `stash-box.key` respectively, and place them in the same directory as stash-box. When Stash-box detects these files, it will use HTTPS instead of HTTP.
 
 ## PHash Distance Matching
```
Comment (DogmaDragon) [inline README.md]: Would change it to `## pHash distance matching` to keep capitalization consistent with other headings.

Comment (Leopere) [reply README.md]: Resolved in next commit.

File: README.md (hunk 1 above)
Comment (Leopere) [reply README.md:5]: The introduction aimed for something a bit more human digestible of you want to propose an additional PR that re-includes this I can merge it but I think a high level abstraction is a bit more digestible.  Doesn't mean I want it gone but I think on this PR I'm going to leave it unless theres a suggestion.

**Review by Leopere — COMMENTED** [review]
Been reviewed by @DogmaDragon and adjustments made by @Leopere

**Review by peroo — APPROVED** [review]

**Pull request merged**